mucommander-core/src/main/java/com/mucommander/ui/main/SplashScreen.java

```diff
@@ -21,16 +21,27 @@
 import com.mucommander.commons.util.ui.dialog.DialogToolkit;
 import com.mucommander.ui.icon.IconManager;
 
-import javax.swing.*;
-import java.awt.*;
-
+// if we want SplashScreen to load faster,
+// then don't load everything from awt.* or swing.* (i.e. no wildcards)
```

Comment:
no need for this comment - we should avoid imports with wildcards anyway as our formatter doesn't accept it

Reply:
done in https://github.com/mucommander/mucommander/pull/1057